src/twitter.ts

```diff
@@ -0,0 +1,161 @@
+import axios from "axios";
+import { OpenSeaCollection } from "opensea-js/lib/types";
+import { TwitterApi } from "twitter-api-v2";
+import { config } from "./config";
+import { openSeaPort } from "./opensea";
+import { PartyEvent } from "./types";
+import {
+  bestUserName,
+  getIsNewPartyWithContribution,
+  // getIsPartyHalfWay,
+} from "./utils";
+
+/**
+ * queries opensea api for the nft's opensea colleciton
+ * @param {PartyEvent} event
+ * @returns {Promise<OpenSeaCollection>}
+ */
+const getOpenseaCollection = async (
+  event: PartyEvent
+): Promise<OpenSeaCollection> => {
```

Comment:
not necessary for merge: but i'm getting a little worried about this returning undefined and us not catching us in the fn that calls this.  

would prefer to be explicit that this is `Promise<OpenSeaCollection | undefined>` and then set `strictNullChecks` to `true` in our tsconfig

Reply:
i think i'd prefer to do this in another pr. this one has grown quite a bit already.

Reply:
makes sense